backend/.env.example

```
@@ -8,4 +8,8 @@ SPOTIFY_CLIENT_ID=
 SPOTIFY_CLIENT_SECRET=
 JWT_SECRET=
 BCRYPT_SALT_ROUNDS=10
- 
\ No newline at end of file
+NCP_ACCESS_KEY=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NCP_ACCESS_KEY key should go before the POSTGRES_DB key

(UnorderedKey)